index.js

```diff
@@ -1,11 +1,23 @@
-// ul要素を取得
 const ul = document.getElementById('ul');
 
 // liタグを作成
 const listItem = document.createElement('li');
+// aタグを作成
+const anker = document.createElement('a');
+// aタグに属性追加
+anker.setAttribute('href','1.html');
+// imgタグを作成
+const img = document.createElement('img');
+// imgタグに属性追加
+img.setAttribute('src','bookmark.png');
+img.setAttribute('alt','ブックマーク');
 
-// liタグにテキストを挿入
-listItem.innerHTML = 'これです';
+// imgタグをaタグの子要素として挿入
+anker.textContent = 'これです';
+anker.insertBefore(img, anker.firstChild);
```

Comment:
`prepend`も同じ動きをします！ご参考程度に。
```suggestion
anker.prepend(img);
```